examples/examples_test.go

```diff
@@ -69,31 +69,26 @@ func validateObject(obj runtime.Object) (errors []error) {
 	return errors
 }
 
-func walkJSONFiles(inDir string, fn func(name, path string, data []byte)) error {
-	err := filepath.Walk(inDir, func(path string, info os.FileInfo, err error) error {
-		if err != nil {
-			return err
-		}
-		if info.IsDir() && path != inDir {
-			return filepath.SkipDir
-		}
-		name := filepath.Base(path)
-		ext := filepath.Ext(name)
-		if ext != "" {
-			name = name[:len(name)-len(ext)]
-		}
-		if !(ext == ".json" || ext == ".yaml") {
-			return nil
-		}
-		glog.Infof("Testing %s", path)
-		data, err := ioutil.ReadFile(path)
-		if err != nil {
-			return err
+func walkJSONFiles(inDir string, fn func(name string, path string, data []byte)) error {
+	return filepath.Walk(inDir, func(path string, info os.FileInfo, err error) error {
+		if err == nil {
```

Comment:
The old way (handling the error first) is actually the go style; see https://code.google.com/p/go-wiki/wiki/CodeReviewComments#Indent_Error_Flow